frontend/src/components/DisplayOffers/DisplayOffers.jsx

```diff
@@ -1,15 +1,16 @@
 import OfferCard from "@components/OfferCard/OfferCard";
 import { useState, useEffect, useContext } from "react";
-import axios from "axios";
+import useApi from "@services/useApi";
 import latinize from "latinize";
 import Style from "./style";
 import { Context } from "../../contexts/Context";
 
 export default function DisplayOffers() {
   const [offers, setOffers] = useState([]);
   const { searchValue } = useContext(Context);
+  const api = useApi();
   useEffect(() => {
-    axios.get(`${import.meta.env.VITE_BACKEND_URL}/offers`).then(({ data }) => {
+    api.get(`${import.meta.env.VITE_BACKEND_URL}/offers`).then(({ data }) => {
```

Comment:
`api.get("/offers")` is enough (<3 this hook)

Reply:
done!